src/rust/Cargo.toml

```diff
@@ -127,6 +127,7 @@ features = [
   "search_sorted",
   "serde-lazy",
   "sign",
+  "sql",
```

Comment:
Is this intentional change?
The polars/sql feature is already included in the sql feature.

Reply:
I thought it would fix the error in the docs CI, I can't reproduce it locally. I'll remove this.

Reply:
The error message seems come from here.
https://github.com/pola-rs/r-polars/blob/7bbe2c69fcad744a5442b4cff4ae4e071f2230d6/R/dataframe__frame.R#L74

IIUC, this line can changed to be `NULL`

Reply:
Thanks, nice catch! Unfortunately there's still one error I can't track down though